dbt/models/eia923/out_eia923__boiler_fuel/schema.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
```

Comment:
@zaneselvans do you know why `out_eia923__boiler_fuel` isn't using the `monthly` naming convention?

Reply:
There are 3 `out_eia923__*_boiler_fuel` tables -- this one hasn't been aggregated by month, so there are probably some cases in which there are multiple records for a single month that get combined in the `__monthly` version. But I'd need to compare the monthly and unaggregated tables to be sure. I know there's at least one case in the generation tables where the monthly and unaggragated tables are literally identical.